frontend/src/pages/talks/TalkEntryList.tsx

```diff
@@ -0,0 +1,19 @@
+import { Box } from '@mui/material';
+import React from 'react';
+
+import TalkSingleEntry from 'src/pages/talks/TalkSingleEntry';
+import { TalkEntry } from 'src/services/mocks';
+
+const TalkEntryList = ({ talks }: { talks: Array<TalkEntry> }) => {
+  return (
+    <>
+      {talks.map((talk) => (
+        <Box key={talk.uid} mx={1} my={2}>
```

Comment:
The horizontal margin is not required here, as the parent component already has an horizontal padding to display the content nicely on each screen size.

```suggestion
        <Box key={talk.uid} my={2}>
```

Reply:
it's remove, thanks